testing/content/DigitalContentShape.ttl

```diff
@@ -125,7 +125,25 @@ shapes:DigitalContentShape
 		sh:severity sh:Violation ;
 		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/DigitalContentShape.ttl> (DigitalContentShape): An ids:accrualPeriodicity property must point from an ids:DigitalContent to an xsd:duration."@en ; 
 	] ;
+    
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:created ;
+		sh:datatype xsd:dateTimeStamp ;
+        sh:maxCount 1 ;
+		sh:severity sh:Violation ;
+		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/DigitalContentShape.ttl> (DigitalContentShape): An ids:DigitalContentShape must have at most one ids:created attribute pointing to a valid xsd:dateTimeStamp."@en ; 
+	] ;
 
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:modified ;
+        sh:maxCount 1 ;
+		sh:datatype xsd:dateTimeStamp ;
+		sh:severity sh:Violation ;
+		sh:message "<https://raw.githubusercontent.com/IndustrialDataSpace/InformationModel/master/testing/content/DigitalContentShape.ttl> (DigitalContentShape): An ids:modified property of an ids:DigitalContent must have valid xsd:dateTimeStamp values as its object."@en ; 
```

Comment:
Here as well. Rather have a comment similar to the one for ids:created

Reply:
Resolved in dbed026